api/api.go

```diff
@@ -85,37 +87,24 @@ type ScannerStats struct {
 	RecordsMatched int64 `json:"records_matched"`
 }
 
-type Pool struct {
-	ID   ksuid.KSUID `json:"id" zng:"id"`
-	Name string      `json:"name" zng:"name"`
-}
-
-type PoolInfo struct {
-	Pool
-	Span *nano.Span `json:"span,omitempty"`
-	Size int64      `json:"size" unit:"bytes"`
-}
-
 type VersionResponse struct {
 	Version string `json:"version"`
 }
 
-// XXX This should use order.Layout.  See #2654.
 type PoolPostRequest struct {
-	Name   string      `json:"name"`
-	Keys   field.List  `json:"keys"`
-	Order  order.Which `json:"order"`
-	Thresh int64       `json:"thresh"`
+	Name   string       `json:"name"`
+	Layout order.Layout `json:"layout"`
+	Thresh int64        `json:"thresh"`
 }
 
 type PoolPutRequest struct {
 	Name string `json:"name"`
 }
 
 type CommitRequest struct {
-	Commit  string `json:"commit"`
-	User    string `json:"user"`
-	Message string `json:"message"`
+	Date    nano.Ts `zng:"date"`
+	Author  string  `zng:"author"`
+	Message string  `zng:"message"`
```

Comment:
Might as well match the order in `lake/commit/actions.CommitAction`.
```suggestion
	Author  string  `zng:"author"`
	Date    nano.Ts `zng:"date"`
	Message string  `zng:"message"`
```